Sources/Sentry/Profiling/SpectoProtoPolyfills.mm

```diff
@@ -0,0 +1,40 @@
+//
+//  SpectoProtoPolyfills.cpp
+//  Sentry
+//
+//  Created by Andrew McKnight on 1/20/22.
+//  Copyright © 2022 Sentry. All rights reserved.
+//
+
```

Comment:
`l`: We don't have copyrights for our code in the code base except for code that we copied from other open source SDKs. Of course, we can keep it, but just want to point it out.

Reply:
this is something we had in a precommit hook, we should look into that for this project, unless you all have custom Xcode file templates you use to avoid these creeping in.